pkg/ccl/importccl/read_import_pgdump.go

```diff
@@ -383,8 +383,10 @@ func readPostgresStmt(
 			case *tree.AlterTableSetDefault:
 				for i, def := range create.Defs {
 					def, ok := def.(*tree.ColumnTableDef)
+					// If it's not a column definition, or the column name doesn't match,
+					// we're not interested in this column.
 					if !ok || def.Name != cmd.Column {
-						return errors.Errorf("unsupported %T definition: %s", def, def)
+						continue
```

Comment:
would it make sense to have a `found` bool and error out if we never find a def with name == cmd.Column? or just assume it won't happen?

Reply:
yeah, done!